scipy/optimize/_minimize.py

```diff
@@ -14,7 +14,6 @@
 
 import numpy as np
 
-
```

Comment:
2 spaces is recommended PEP8, so this was probably an accident. doesn't really matter, IIRC both are now allowed